.github/renovate.json

```diff
@@ -0,0 +1,62 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base"
+  ],
+  "prConcurrentLimit": 5,
+  "baseBranches": ["master","release-1.8","release-1.9","release-1.10"],
+  "ignorePaths": ["design/**"],
+  "postUpdateOptions": ["gomodTidy"],
+  "recreateClosed": true,
+  "regexManagers": [
+    {
+      "fileMatch": ["^.github/workflows/ci.yml$"],
```

Comment:
We have a few other workflows that we'll also want to keep consistent on Go versions, such as https://github.com/crossplane/crossplane/blob/3b22fcba51b1a445146cce539a338d06f6ee9a79/.github/workflows/promote.yml#L16

Reply:
I'll grep around the repo to be sure to handle all of them 👍